flag.go

```diff
@@ -138,6 +138,9 @@ type FlagSet struct {
 	// help/usage messages.
 	SortFlags bool
 
+	// IgnoreUnknownFlags is used to indicate to ignore unknown flags
+	IgnoreUnknownFlags bool
```

Comment:
Convert this into some sort of generic "set" of error handling bools which we can grow as needed and I'll merge this PR.

Reply:
Will do that today, thanks